cfgov/cfgov/settings/base.py

```diff
@@ -0,0 +1,89 @@
+import os
+from unipath import Path
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+CFGOV_REFRESH = Path(__file__).ancestor(5).child('cfgov-refresh')
```

Comment:
Also concerning [my other comment](), this could be `CFGOV_REFRESH = Path(__file__).ancestor(5).child('cfgov-refresh').child('dist')` to solve that problem.